skimage/registration/_affine.py

```diff
@@ -0,0 +1,201 @@
+                                           order=1)
+    else:
+        transformed = np.zeros_like(moving_image)
+        for ch in range(moving_image.shape[-1]):
+            ndi.affine_transform(moving_image[..., ch], transformation,
+                                 order=1, output=transformed[..., ch])
+    return cost(reference_image, transformed)
+
+
+def affine(reference_image, moving_image, *,
+           cost=cost_nmi, initial_vector=None,
+           translation_indices=None, vector_to_matrix=None,
+           pyramid_scale=2, pyramid_minimum_size=8, multichannel=False,
+           inverse=True, level_callback=None, method='Powell',
+           **kwargs):
+    """Find a transformation matrix to register a moving image to a reference.
+
+    Parameters
+    ----------
+    reference_image : ndarray
+        A reference image to compare against the target.
+    moving_image : ndarray
+        Our target for registration. Transforming this image using the
+        returned matrix aligns it with the reference.
+    cost : function, optional
+        A cost function which takes two images and returns a score which is
+        at a minimum when images are aligned. Uses the normalized mutual
+        information by default.
+    initial_vector : array of float, optional
+        The initial vector to optimize. This vector should have the same
+        dimensionality as the transform being optimized. For example, a 2D
+        affine transform has 6 parameters. A 2D rigid transform, on the other
+        hand, only has 3 parameters.
+    translation_indices : array of int, optional
+        The location of the translation parameters in the parameter vector. If
+        None, the positions of the translation parameters in the raveled
+        affine transformation matrix, in homogeneous coordinates, are used. For
+        example, in a 2D transform, the translation parameters are in the
+        top two positions of the third column of the 3 x 3 matrix, which
+        corresponds to the linear indices [2, 5].
+    vector_to_matrix : callable, array (M,) -> array-like (N+1, N+1), optional
+        A function to convert a linear vector of parameters, as used by
+        `scipy.optimize.minimize`, to an affine transformation matrix in
+        homogeneous coordinates.
+    pyramid_scale : float, optional
+        Scaling factor to generate the image pyramid.
+    pyramid_minimum_size : integer, optional
+        The smallest size for an image along any dimension. This value
+        determines the size of the image pyramid used. Choosing a smaller value
+        here can cause registration errors, but a larger value could speed up
+        registration when the alignment is easy.
+    multichannel : bool, optional
+        Whether the last axis of the image is to be interpreted as multiple
+        channels or another spatial dimension. By default, this is False.
+    inverse : bool, optional
```

Comment:
Mention default value.

Reply:
This goes against our usual policy, but I agree that this one is important enough to mention.